adserver/views.py

```diff
@@ -414,6 +414,7 @@ def get(self, request, advertisement_id, nonce):
         """Handles proxying ad views and clicks and collecting metrics on them."""
         advertisement = get_object_or_404(Advertisement, pk=advertisement_id)
         publisher = advertisement.get_publisher(nonce)
+        div_id, keywords, ad_type = advertisement.get_request_data(nonce)
```

Comment:
Wondering if it makes sense to just use this as `request_data` and pass it along to each of the functions as that single argument? Might be cleaner.